source/game_sa/CarAI.cpp

```diff
@@ -42,7 +42,13 @@ void CCarAI::InjectHooks() {
 
 // 0x41C4A0
 void CCarAI::AddAmbulanceOccupants(CVehicle* vehicle) {
-    plugin::Call<0x41C4A0, CVehicle*>(vehicle);
+    CPed* driver    = vehicle->SetUpDriver(PED_TYPE_NONE, false, false);
+    CPed* passenger = vehicle->SetupPassenger(1, PED_TYPE_NONE, false, false);
+
+    driver->GetTaskManager().SetTask(new CTaskSimpleCarDrive{ vehicle, nullptr, false }, TASK_PRIMARY_DEFAULT);
+    driver->GetTaskManager().SetTask(new CTaskComplexMedicTreatInjuredPed{ vehicle, passenger, true }, TASK_PRIMARY_PRIMARY);
+    passenger->GetTaskManager().SetTask(new CTaskSimpleCarDrive{ vehicle, nullptr, false }, TASK_PRIMARY_DEFAULT);
+    passenger->GetTaskManager().SetTask(new CTaskComplexMedicTreatInjuredPed{ vehicle, driver, false }, TASK_PRIMARY_PRIMARY);
```

Comment:
```suggestion
    driver->GetTaskManager().SetTask(new CTaskSimpleCarDrive{ vehicle }, TASK_PRIMARY_DEFAULT);
    driver->GetTaskManager().SetTask(new CTaskComplexMedicTreatInjuredPed{ vehicle, passenger, true }, TASK_PRIMARY_PRIMARY);
    passenger->GetTaskManager().SetTask(new CTaskSimpleCarDrive{ vehicle }, TASK_PRIMARY_DEFAULT);
    passenger->GetTaskManager().SetTask(new CTaskComplexMedicTreatInjuredPed{ vehicle, driver, false }, TASK_PRIMARY_PRIMARY);
```

`CTaskSimpleCarDrive` ctor is defaulted.

```cpp
explicit CTaskSimpleCarDrive(CVehicle* vehicle, CTaskUtilityLineUpPedWithCar* utilityTask = {}, bool updateCurrentVehicle = {});
```

Reply:
these classes aren't included in this file, add this to line 11:

```cpp
#include "TaskComplexMedicTreatInjuredPed.h"
#include "TaskSimpleCarDrive.h"
```

Reply:
Exactly! Thanks will do